tools/kete.py

```diff
@@ -699,6 +704,9 @@ def is_index_or_all(string):
                             type=is_index_or_all,
                             const='all', nargs='?', default='all',
                             help='the index of the drawing to fetch or a literal "all"')
+        parser.add_argument('--format', metavar='{svg|png}',
+                            default='svg',
```

Comment:
please add `choices = ['svg', 'png']` here so argparse takes care of faulty arguments here.

Reply:
Done